docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -0,0 +1,96 @@
+# OpenShift Service Mesh 2.6 Cluster wide --> 3 Migration guide
+This guide is for users who are currently running `ClusterWide` OpenShift Service Mesh 2.6 migrating to OpenShift Service Mesh 3.x. You should first read [this document comparing OpenShift Service Mesh 2 vs. OpenShift Service Mesh 3](../../ossm2-vs-ossm3.md) to familiarize yourself with the concepts between the two versions and the differences in managing the workloads and addons.
+
+## Migrating OpenShift Service Mesh 2.6 Cluster wide to OpenShift Service Mesh 3
+
+### Prerequisites
+- you have completed all the steps in the [pre-migration checklist](../README.md#pre-migration-checklist)
+- you have read [OpenShift Service Mesh 2 vs. OpenShift Service Mesh 3](../../ossm2-vs-ossm3.md)
+- you have verified that your 2.6 `ServiceMeshControlPlane` is using `ClusterWide` mode
+- Red Hat OpenShift Service Mesh 3 operator is installed
+- `IstioCNI` is installed
+- `istioctl` is installed
+
+### Procedure
+In this example, we'll be using the [bookinfo demo](https://raw.githubusercontent.com/Maistra/istio/maistra-2.6/samples/bookinfo/platform/kube/bookinfo.yaml) but you can follow these same steps with your own workloads.
+
+TODO: should we add a note about `minReplicas` for workload deployments?
+TODO: include also ingress gateway deployment migration?
+
+#### Plan the migration
+There will be two cluster wide Istio control planes running during the migration process so it's necessary to plan the migration steps in advance in order to avoid possible conflicts between the two control planes.
+
+There is a few conditions which must be assured to achieve the seamless migration:
+- both control planes must share the same root certificate
+
+  This can be achieved simply by installing the 3.0 control plane to the same namespace as 2.6 control plane
+- both control planes have access to all migrated namespaces and namespaces running addons (e.g. otel collector)
+  1. verify there are no Network Policies blocking the traffic
+  1. `discoverySelectors`  defined in 3.0 `Istio` resource must match migrated namespaces. See [Scoping the service mesh with DiscoverySelectors](../../create-mesh/README.md)
```

Comment:
Is this only applicable if you are running multiple controlplanes in cluster wide mode?

Reply:
I think I need to update this so it clearer. What I mean by this is: if you decide to use discovery selectors for 3.0, you have to assure, that 3.0 control plane will have access to all 2.6 namespaces which you want to migrate - so 3.0 control plane internal service registry is the same as 2.6 internal service registry